components/brave_extension/extension/brave_extension/manifest.json

```diff
@@ -22,26 +22,7 @@
     ],
     "persistent": true
   },
-  "externally_connectable": {
```

Comment:
was implemented here: https://github.com/brave/brave-core/commit/0e118d9ee75691b621095cdf627c913fe29ea19a

@petemill says that we don't use it for Brave News anymore.